contributors/guide/README.md

```diff
@@ -168,9 +168,17 @@ _Improvements needed_
 
 ## Testing
 
-_Improvements needed_
-* link to testing process 
-* walkthrough of where to find what in the tests (how to use, how to debug)
+Testing is the responsibility of all contributors and is in part owned by all sigs, but is also coordinated by [sig-testing](https://github.com/kubernetes/community/tree/master/sig-testing).
+
+The main testing overview document is [here](https://github.com/kubernetes/community/blob/master/contributors/devel/testing.md).
+
+There are three coarse types of test in kubernetes.  The location of the test code varies with type, as does the specifics of the environment needed to successfully run the test:
+
+* Unit: These confirm that a particular function behaves as intended.  Golang includes native ability for unit testing via the [testing](https://golang.org/pkg/testing/) package.  Unit test source code can be found adjacent to the corresponding source code within a given package.  For example: functions defined in [kubernetes/cmd/kubeadm/app/util/version.go](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/version.go) will have unit tests in [kubernetes/cmd/kubeadm/app/util/version_test.go](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/version_test.go).  These are easily run locally be any developer on any OS.
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/version.go should be https://git.k8s.io/kubernetes/cmd/kubeadm/app/util/version.go

Reply:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/version_test.go should be https://git.k8s.io/kubernetes/cmd/kubeadm/app/util/version_test.go

Reply:
fixed